tests/AutoMapperBaseTest.php

```diff
@@ -33,7 +33,7 @@ protected function setUp(): void
         $this->buildAutoMapper();
     }
 
-    protected function buildAutoMapper(bool $allowReadOnlyTargetToPopulate = false): AutoMapper
+    protected function buildAutoMapper(bool $allowReadOnlyTargetToPopulate = false, bool $mapPrivatePropertiesAndMethod = true, string $classPrefix = 'Mapper_'): AutoMapper
```

Comment:
Shouldn't we put `$mapPrivatePropertiesAndMethod` at `false` by default ? (Like it is in `AutoMapper` class static method)

Reply:
I think you're right, I leaved this to `true` because it was easier 😅
But I don't even think that much tests will fail if I change this

Reply:
```ERRORS!
Tests: 167, Assertions: 455, Errors: 6, Failures: 10.
```